src/screens/editor/container/editorContainer.js

```diff
@@ -101,6 +101,7 @@ class EditorContainer extends Component {
     ImagePicker.openPicker({
       includeBase64: true,
       multiple: true,
+      smartAlbums: ['UserLibrary', 'Favorites', 'PhotoStream', 'Panoramas', 'Bursts'],
```

Comment:
We should probably add `mediaType: 'photo'`
`mediaType | string (default any) | Accepted mediaType for image selection, can be one of: 'photo', 'video', or 'any'`

Reply:
agreed: because smartAlbums list only works for iOS... mediaType will cover android too